src/registrar/views/domain.py

```diff
@@ -96,6 +106,48 @@ def form_valid(self, formset):
         return super().form_valid(formset)
 
 
+class DomainYourContactInformationView(DomainPermission, FormMixin, DetailView):
+
+    """Domain your contact information editing view."""
+
+    model = Domain
+    template_name = "domain_your_contact_information.html"
+    context_object_name = "domain"
+    form_class = ContactForm
+
+    def get_form_kwargs(self, *args, **kwargs):
+        """Add domain_info.submitter instance to make a bound form."""
+        form_kwargs = super().get_form_kwargs(*args, **kwargs)
+        form_kwargs["instance"] = self.get_object().domain_info.submitter
+        return form_kwargs
+
+    def get_success_url(self):
+        """Redirect to the your contact information for the domain."""
+        return reverse("domain-your-contact-information", kwargs={"pk": self.object.pk})
+
+    def post(self, request, *args, **kwargs):
+        """Form submission posts to this view."""
+        self.object = self.get_object()
+        form = self.get_form()
+        if form.is_valid():
+            # there is a valid email address in the form
```

Comment:
Is this comment correct? It implies that is_valid only checks if the email is valid, but first/lastname etc are also set as required on the contact page.